src/internal/data/data_collection_test.go

```diff
@@ -0,0 +1,57 @@
+package data
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/alcionai/corso/src/pkg/path"
+)
+
+func TestStateOf(t *testing.T) {
```

Comment:
we've been trying to have all tests use testify. can you please make a testify test suite for this and run it with that?
```suggestion
type DataCollectionSuite struct {
  suite.Suite
}

func TestDataCollectionSuite(t *testing.T) {
  suite.Run(t, new(DataCollectionSuite)
}

func (suite *DataCollectionSuite) TestStateOf(t *testing.T) {
```

Reply:
Resolved in https://github.com/alcionai/corso/pull/2455